docs/topics_5/web-custom-rules.adoc

```diff
@@ -4,48 +4,69 @@
 = Using custom rules and labels
 
 [id='adding_custom_rules_{context}']
-== Adding custom rules
+== Adding global custom rules
 
-You can add your own custom rules to use during analysis of your applications.
+You can add global custom rules to use during analysis of all your applications.
 For information on writing custom {ProductShortName} rules, see the {ProductShortName} link:{ProductDocRulesGuideURL}[_{RulesDevBookName}_].
 
-Select *Rules Configuration* from the *User* menu in the title bar to view all existing system and custom rules.
+Click *Rules Configuration* in the navigation bar to view all existing system and custom rules.
 
 .Adding custom rules
-image::web-rules-config.png[Add project]
+image::web-rules-config.png[Add custom rules]
 
-To add your own custom rules, click *Add*. You can upload rules or register a server path that contains the rules.
+You can upload rules or register a server path that contains the rules.
 
-Uploading rules::
+.Procedure
 
-You can use the *Choose Files* button to select ruleset files, or you can drag and drop the files into the area provided. This uploads the selected ruleset files to the {ProductShortName} server. Click *Add* to upload the rules.
+Click *Add new*.
 
-Registering a server path::
+A window containing two tabs, *Upload* and *Server path* opens.
+.. In the *Upload* tab, click *Browse* and then select a rule.
+.. In the *Server path* tab, enter the *Rules path* of the server-side file or directory containing the rules in the text box, select *Scan recursively* if relevant, and then click *Save*.
 
-Select the *Server Path* tab and enter a path on the server that contains rules to be available to projects. This does not copy the files to the {ProductShortName} server, but instead uses the rules found in the specified location during analysis. This way, if the rules are updated in this location, the {ProductShortName} server will always use the latest versions of rules during analysis. Click *Add* to register the rules.
+== Viewing rule details
 
-Custom rulesets may be selected individually during a project analysis. To enable these select the ruleset in the *Use local custom rules* section of the analysis configuration.
+Rules are listed in a table in the *Rules configuration* page. Each row in the table displays a rule's short path, source, target, and number of rules. You can also view each rule's details.
+
+.Procedure
+
+Right-click the kebab icon in the rule’s row and then click *View details* in the list that opens. The rule's details are displayed. If the short path points to more than one one rule, you are prompted to select one rule from the list that opens.
```

Comment:
"Right-click the kebab icon in the rule’s row and then click *View details* in the list that opens."

Are you sure it's a 'right-click'? That's unusual in Patternfly. Please include an icon screenshot. It's not called kebab in formal docs, btw. It's usually "Options icon". 

'rule' is inanimate, so no apostrophe-s.

I'm guessing that it should be: "Click the Options icon (image) and select "View details". 
You probably don't need "The rule's details are displayed." because the user can see that.

"If the short path points to more than one one rule, you are prompted to select one rule from the list that opens."

What is "prompted"? What does that mean?
Fix the "one one" typo.
"list that opens" is awkward. You don't need it.